### DISCUSSION TITLE
Using Github Wiki with Github Desktop

### DISCUSSION THREAD

**Discussion by joostschouppe** [community/community#23720] [New to GitHub]
<p>When I try to open any Github Wiki pages in Github Desktop (with the URL provided on the web of  <code>https://github.com/organization_name/repo_name.wiki.git</code> ) , I get an error: “We couldn’t find the repository”. That’s kind of weird, since it’s the URL provided on the web, format is <code>https://github.com/organization_name/repo_name.wiki.git</code></p>
<p>I suppose it’s just a wrong error message? Or is there some way to do this properly?</p>
<p>(I wouldn’t mind using plain .md files for my documentation error, but would like to have the online Markdown editor available while editing)</p>

**Comment by nethgato** [comment, marked as answer]
<p>Hey <a class="mention" href="/joostschouppe">@joostschouppe</a> o/</p>
<p>Unfortunately, this is a known issue:</p>
<aside class="onebox githubissue">
  <header class="source">
      <a href="https://github.com/desktop/desktop/issues/3839" target="_blank" rel="noopener">github.com/desktop/desktop</a>
  </header>
  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Issue">
	  <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/desktop/desktop/issues/3839" target="_blank" rel="noopener">GitHub wiki repos not detected correctly, asks user to publish again</a>
    </h4>

    <div class="github-info">
      <div class="date">
        opened <span class="discourse-local-date" data-format="ll" data-date="2018-01-22" data-time="03:22:16" data-timezone="UTC">03:22AM - 22 Jan 18 UTC</span>
      </div>


      <div class="user">
        <a href="https://github.com/brcontainer" target="_blank" rel="noopener">
          <img alt="brcontainer" src="https://user-images.githubusercontent.com/62281706/181078907-b4da9273-0730-4d9f-9315-94999873e9a1.jpeg" class="onebox-avatar-inline" width="20" height="20">
          brcontainer
        </a>
      </div>
    </div>
  </div>
</div>

<div class="github-row">
  <p class="github-content">Description
After clone a wiki repository and close GithubDesktop can no longer identify the origin.
Version
GitHub Desktop version: 1.0.12 and 1.0.13
git rev-parse HEAD:...</p>
</div>

<div class="labels">
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">bug</span>
    <span style="display:inline-block;margin-top:2px;background-color: #B8B8B8;padding: 2px;border-radius: 4px;color: #fff;margin-left: 3px;">priority-3</span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>…but I highly encourage you to participate in the issue above.</p>
